### ISSUE TITLE
Could I add the demo of CppCon2018 to the llvm test suite ?

### ISSUE THREAD

**Issue by ChuanqiXu9** [GorNishanov/await#4]
Hi Gor,

Now the llvm lacks a benchmark to measure the performance for coroutines. I think this one is relatively suit for that.
And I am worrying about the license problem. Do you feel comfortable if I submit the code to llvm test suite (I would mark you as the author).

**Comment by GorNishanov** [comment]
Hi Chuanqi:

I am not sure that this particular benchmark is good for automated test suite. When we are measuring differences related to memory latency, it is too fine timing for repeatable tests and reliable performance tests.

**Comment by ChuanqiXu9** [comment]
Yeah, maybe it is too fine and the result number may change surprisingly. But I think that one is much better than none. May be it is not the best. I believe it would be helpful.
